packages/plugins/IconMenu/src/components/IconMenu.vue

```diff
@@ -112,31 +98,33 @@ export default Vue.extend({
 
 <style lang="scss" scoped>
 .icon-menu-list {
+  position: relative;
   list-style: none;
+  padding: 0;
 }
 
 .icon-menu-list-item-horizontal {
-  position: relative;
   float: left;
   margin-left: 0.5rem;
 }
 
 .icon-menu-list-item {
-  position: relative;
   margin-bottom: 0.5rem;
   z-index: 1;
 }
 
 .icon-menu-list-item-content {
   position: absolute;
   white-space: nowrap;
-  right: calc(100% + 0.5rem);
+  top: 0;
+  right: calc(100% + 0.5em);
 }
 
 .icon-menu-list-item-content-horizontal {
   position: absolute;
   white-space: nowrap;
-  top: 3em;
+  top: calc(100% + 0.5em);
+  right: -0.5em;
```

Comment:
```suggestion
  right: 0;
```
Otherwise it hugs the side of the display

Reply:
I figured it was more important to have the box align to the buttons than to have the sidebar not hugging the side of the viewport.

![spacing](https://github.com/Dataport/polar/assets/108349707/dd2bc199-ab5c-4407-8815-381f5b134527)
![spacing-with-sidebar](https://github.com/Dataport/polar/assets/108349707/00642547-2f45-49c7-b4f5-a720b6a8cf4d)

Reply:
When testing with `right: 0;` on a phone with a viewport of `412px x 919px`, I received the following results in Chrome:

![att_0](https://github.com/Dataport/polar/assets/73653210/f4da3b11-41fe-443e-8424-eff94722860e)
![layerchooser_0](https://github.com/Dataport/polar/assets/73653210/5d6dec08-a255-47bc-b21d-ad14cb7a68b4)

With `right: -0.5em;` the results were the following:

![att_-0 5em](https://github.com/Dataport/polar/assets/73653210/e0e4b52a-6f2b-4f5b-bc9d-1bd9703a8216)
![layerchooser_-0 5em](https://github.com/Dataport/polar/assets/73653210/515ac499-5b26-460e-9534-ae3da96aeb74)

Meaning, that the alignment with the Buttons was only present when using `right: 0;`.

Could you please elaborate in which use cases the above situations would occur?

Reply:
`right: 0;` visibly doesn't produce alignment between the buttons and the contents in your screenshots.

My screenshots have been provided from desktop browsers. I see, the difference is that `scrollbar-gutter` produces the right distance in desktop browsers, but not on mobile devices, where scrollbars are usually these thin lines atop the inside of boxes.

For the time being, I suggest we leave it at that. It looks good on desktop and acceptable on mobile devices. Let's say it's intentional to make better usage of the available space. With `right: 0;`, both are slightly off.

Reply:
We'll probably get back to this, but fine by me for the time being